fern/pages/cpp_api/cpp-api-cluster-kmeans.md

```
@@ -0,0 +1,1150 @@
+| `sample_weight` | in | `std::optional<raft::device_vector_view<const double, int64_t>>` | Optional weights for each observation in X. [len = n_samples] |
+| `centroids` | inout | `raft::device_matrix_view<double, int64_t>` | [in] When init is InitMethod::Array, use centroids as the initial cluster centers. [out] The generated centroids from the kmeans algorithm are stored at the address pointed by 'centroids'. [dim = n_clusters x n_features] |
+| `inertia` | out | `raft::host_scalar_view<double>` | Sum of squared distances of samples to their closest cluster center. |
+| `n_iter` | out | `raft::host_scalar_view<int64_t>` | Number of iterations run. |
+
+**Returns**
+
+`void`
+
+_Source: `cpp/include/cuvs/cluster/kmeans.hpp:430`_
+
+### cuvs::cluster::kmeans::fit
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)